content/en/tracing/trace_collection/trace_context_propagation/cpp.md

```diff
@@ -13,77 +13,9 @@ further_reading:
       text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
 ---
 
-## Overview
+This page documents additional use cases of trace context propagation for the Datadog C++ SDK. For a high-level overview of trace context propagation, see [Trace Context Propagation][1].
```

Comment:
Rather than having a page here, can we put this into a collapsable section within the CPP tab of the main page? The same goes for .NET and PHP

```
{{% collapse-content title="Additional Use Cases" level="h4" %}}
Content goes here
{{% /collapse-content %}}
```

Reply:
Great idea, will do!

Reply:
Done in the following commits:
- cc946ea56b14ea55d7053b90ba0175646535fbd6
- 80b9f4a42b0c8a92cd1ef332965613811ca4ccc8
- f13ef990ddaac9ec22a705ed55a42363d55b51d3